packages/go/analysis/post.go

```diff
@@ -134,9 +134,9 @@ func DeleteTransitEdges(ctx context.Context, db graph.Database, fromKind, toKind
 		if err := db.ReadTransaction(ctx, func(tx graph.Transaction) error {
 			fetchedRelationshipIDs, err := ops.FetchRelationshipIDs(tx.Relationships().Filterf(func() graph.Criteria {
 				return query.And(
-					query.Kind(query.Start(), fromKind),
+					query.KindIn(query.Start(), baseKinds...),
```

Comment:
This is a bit of a tangent but is there reason we have both query.Kind and query.KindIn? Does the other serve as an alias? It looks like they do the same thing and I was curious what drove the update here

Reply:
The AST that each function builds is different so I wanted distinction here, however, I agree with you this ends up feeling pretty obtuse. It might be worth just making `query.Kind()` variadic.

Unfortunately both `Kind` and `KindIn` are heavily used and would not be a simple refactor. I'll create a ticket to track it for continuity work.

Reply:
Yea, similarly I've been wanting to update the definition of "admin_tier_0" to put into the common schema but it's used in a lot of places to update in a non related changeset without being noisy. FWIW I believe query.Kind and query.KindIn used to function differently in the past.